src/BoilerPlate/Entity.cpp

```diff
@@ -0,0 +1,62 @@
+#include "Entity.hpp"
+
+const float INITIAL_WINDOW_WIDTH = 1136.0f;
+const float INITIAL_WINDOW_HEIGHT = 640.0f;
+
+Entity::Entity()
+{
+	entityPosition = new Vector2(Vector2::Origin);
+	entityOrientation = 0.0f;
+
+	minWindowHeight = -INITIAL_WINDOW_HEIGHT / 2;
+	maxWindowHeight = -minWindowHeight;
+	minWindowWidth = -INITIAL_WINDOW_WIDTH / 2;
+	maxWindowWidth = -minWindowWidth;
+}
+
+void Entity::DrawEntity()
+{
+	glBegin(GL_LINE_LOOP);
+	for (std::vector<Vector2>::iterator it = entityPoints.begin();
+		it != entityPoints.end(); it++)
+	{
+		glVertex2f((*it).x, (*it).y);
```

Comment:
Aunque `it` es una sola variable, la operación sobre la misma (la derreferencia) se repite, por lo que crearía una variable para albergar el resultado de esta operación.